config/routes.rb

```diff
@@ -3,5 +3,9 @@
 Rails.application.routes.draw do
   devise_for :users
   resources :repairs
+  
+  get 'volunteers/index'
+  get 'volunteers/new'
```

Comment:
[Style]
The more Rails-y way of writing this would be

```suggestion
  resources :volunteers, only: [:index, :new]
```